### PR TITLE
Re-build human genome annotations with GDC fasta

### LINKED ISSUE #128: set default human genome+annotation combo to match GDC
Match GDC
- hg38
- v36
[ref](https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files)

### LINKED ISSUE #129: Change human reference fasta
Match [GDC](https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files)
- include decoys
- include viruses

Will need to 
- [x] recreate reference files for all human (h38/hg19) genomes + annotations combos on BIOWULF
- [x] move "new" reference files over to FRCE
- [ ] share "new" reference files with SF

### LINKED ISSUE #138: change fasta for hg19
hg19_19 & hg19_36

- fasta from https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_19/GRCh37.p13.genome.fa.gz
- gtf from https://www.gencodegenes.org/human/release_36lift37.html for hg19_36
- add viruses and decoys from GDC https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files

related to #129

### PR DESCRIPTION
**Pull request by kelly-sovacool**
## Changes

- Rebuilt hg19 and hg38 annotations with the GDC reference fasta, which includes additional sequences. These are stored in a shared location for use in other pipelines too: `/data/CCBR_Pipeliner/db/PipeDB/GDC_refs/`
- Copied files to FRCE: `/mnt/projects/CCBR-Pipelines/db/GDC_refs`

See the snakemake workflow here for how these were built: https://github.com/CCBR/build-renee-refs

## Issues

- Resolves #129 
- Resolves #138 
- Resolves #128

## PR Checklist

(~Strikethrough~ any points that are not applicable.)

- [x] This comment contains a description of changes with justifications, with any relevant issues linked.
- [x] Update docs if there are any API changes.
- [x] Update `CHANGELOG.md` with a short description of any user-facing changes and reference the PR number. Guidelines: https://keepachangelog.com/en/1.1.0/

### CODE REVIEW

**Comment by kelly-sovacool** [comment]
Running tests.

```sh
RENEE_REPO=/data/CCBR_Pipeliner/Pipelines/RENEE/renee-dev-sovacool
for genome in $(ls $RENEE_REPO/config/genomes/biowulf/ | grep hg |  sed 's/.json//'); do
    $RENEE_REPO/bin/renee run \
        --input $RENEE_REPO/.tests/*.R1.fastq.gz \
        --genome $genome \
        --mode slurm \
        --output /data/$USER/renee_test_$genome \
        --sif-cache /data/CCBR_Pipeliner/SIFS \
        &> /data/$USER/renee_shell.${genome}.out
done
```


## Job status

<details>
<summary>generate table</summary>

```sh
cd /data/sovacoolkl/
for f in $(ls | grep renee_shell); do genome=$(echo $f |  sed 's/renee_shell\.//' | sed 's/\.out//'); jobid=$(grep master $f | sed 's/.*: //'); echo "| $genome | $jobid |  |"; done
```

</details>

| genome  | slurm id | status |
|---------|----------|--------|
| hg19_19 | 31994530 |    RUNNING  |
| hg19_36lift37 | 31994535 |    RUNNING  |
| hg38_30 | 31994547 |    RUNNING  |
| hg38_34 | 31994560 |    RUNNING  |
| hg38_36 | 31994162 |    RUNNING  |
| hg38_38 | 31994566 |    RUNNING  |
| hg38_41 | 31994570 |    RUNNING  |
| hg38_45 | 31994583 |    RUNNING  |

## Update

All pipeline runs completed successfully. Jobby initially failed because it wasn't in my path when I submitted these, but after running jobby manually I verified that all jobs completed successfully.

File: config/genomes/biowulf/hg19_36lift37.json
Comment (kopardev) [inline config/genomes/biowulf/hg19_36lift37.json:1]: - For simplicity, rename the file to `hg19_36.json`
- add a note in the commit on how the file `GRCh37.p13.genome.d1.vd1.genome.fa` was created... may be we need to add note about this in RENEE docs as well.

File: config/genomes/igenomes/hg19_19.json
Comment (kopardev) [inline config/genomes/igenomes/hg19_19.json:1]: what are these files in under `igenomes`??

**Review by kopardev — CHANGES_REQUESTED** [review]
Please see comments before merging.

**Comment by kelly-sovacool** [comment]
@kopardev this is ready for your re-review

**Review by kopardev — APPROVED** [review]

**Pull request merged**